### ISSUE TITLE
Google API Client Libraries added?

### ISSUE THREAD

**Issue by IzzySoft** [breezy-weather/breezy-weather#1253]
### Steps to reproduce

The scanner at IzzyOnDroid today reported:

    ! org.breezyweather: needs NonFreeNet

Taking a closer look, it seems you've added the [Google API Client Libraries](https://developers.google.com/api-client-library/java/). Not sure if that was on purpose, or if some dependency dragged that in – but could you please check and, if possible, remove this again? Until then, we'll have to add the `NonFreeNet` anti-feature here.

### Expected behavior

lib not there

### Actual behavior

lib there

### Weather source used

repo

### Breezy Weather version

5.2.7

### Android version

n/a

### Device

n/a

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue.
- [X] I checked that my issue is not covered in the [Frequently Asked Questions](https://github.com/breezy-weather/breezy-weather/blob/main/HELP.md) document.
- [X] I have updated the app to **[version 5.2.7 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)** and I can reproduce the issue in this version.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance), unless my issue only happens with a specific language. I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by IzzySoft** [comment]
Err… I should have added a note: duplicate of #1206, sorry…

**Comment by IzzySoft** [comment]
Btw, have permanently removed that entry from the LibRadar (V1 here btw) definitions, so this won't happen again. There are additional signature definitions in place here meanwhile.

> LibRadar source code mentions only once "Google API Client Libraries"

215 hits in `tgst5.dat`: [tgst5.diff.txt](https://github.com/user-attachments/files/16570620/tgst5.diff.txt)
